assets/constants.ts

```diff
@@ -5323,5 +5323,5 @@ export = {
   // Maximum allowed length of a username.
   "MAX_USERNAME_LENGTH": 30,
 
-  "DEV_MODE": true
+  "DEV_MODE": false
```

Comment:
Why this change?

Reply:
Oh the command `python -m scripts.start --no_browser --prod_env` changes it to false ill change it back

Reply:
Done